src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing.mdx

```diff
@@ -39,26 +39,19 @@ Data Plus requires an organization with:
 
 To upgrade to Data Plus, you'll go to the [**Data retention** UI](/docs/data-apis/manage-data/manage-data-retention#adjust-retention) and click **Edit data retention**. If you meet the requirements and don't see the upgrade option, talk to your New Relic account representative. 
 
-Data Plus features include the following. Limitations of the [Azure Native New Relic Service](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native) are below this list. 
+Here are details about Data Plus features and availability: 
 
 * Increased [query limits](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries), upon enablement, of up to one trillion data points per 30 minutes and 100 billion data points per minute (compared to lower limits like 300 billion data points per 30 minutes for our original data option).
 * Increased [max query duration](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#query-duration) of two minutes for a regular query and 10 minutes for an [asynchronous query](/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial). 
 * Increased retention of several important data types ([learn more](/docs/data-apis/manage-data/manage-data-retention)).
 * Our [log obfuscation UI](/docs/logs/ui-data/obfuscation-ui).
-* With [Enterprise edition](https://newrelic.com/pricing): [FedRAMP](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints) and [HIPAA](/docs/security/security-privacy/compliance/hipaa-readiness-new-relic)-compliant data security, for eligible customers.
-* [Historical data export](/docs/apis/nerdgraph/examples/nerdgraph-historical-data-export), which lets you run larger queries with no timeout.
 * [Streaming data export](/docs/apis/nerdgraph/examples/nerdgraph-streaming-export), which lets you stream ingested data to AWS Kinesis.
-* [Vulnerability Management](/docs/vulnerability-management/overview). 
+* With [Enterprise edition](https://newrelic.com/pricing): [FedRAMP](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints) and [HIPAA](/docs/security/security-privacy/compliance/hipaa-readiness-new-relic)-compliant data security, for eligible customers. (Enterprise edition not yet available for [Azure Native New Relic Service](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native).)
```

Comment:
Is the link formatting right for the HIPAA link? Looks like a hanging hyphen but I could be wrong

Reply:
Just a really small thing, for the last sentence should the phrase enclosed in parenthesis be baked in to the preceding sentence? The whole sentence in a parenthesis thing looks a little awkward to me, but it's just something to consider.